packages/webapp/src/pages/election/index.tsx

```diff
@@ -50,6 +65,14 @@ export const ElectionPage = (props: Props) => {
         keepPreviousData: true,
     });
```

Comment:
And I recommend these queries become custom hooks too so we can easily reuse them elsewhere if we want to.